### PR TITLE
Ekmel - w1

### PR DESCRIPTION
**Pull request by ekmkal**

### CODE REVIEW

File: Week1/homework/exercise-2/exercise2.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  password : 'hyfpassword',
+  database : 'new_world'
+});
+ 
+connection.connect();
+
+connection.query('USE new_world');
+
+// 1.The names of countries with population greater than 8 million
+connection.query("SELECT Name FROM country WHERE population > 8e6", (error, results, fields) => {
```
Comment (mogaleaf) [inline Week1/homework/exercise-2/exercise2.js:15]: Nice notation

**Review by mogaleaf — COMMENTED** [review]
Hi Ekmel,

Great homework,
Both of your exercises are good, you even used autom increment.
The code is neat, easy to read.
Well done!

File: Week1/homework/exercise-1/exercise1.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+});
+
+// Delete existing database
+connection.query(("DROP DATABASE IF EXISTS meetup"), (error, results, fields) => {
+    if (error) throw error;
+    console.log("Database deleted.");
+});
+
+// Create new database
+connection.query(("CREATE DATABASE meetup"), (error, results, fields) => {
```
[the remark this answers is not in the document]
Comment (ekmkal) [reply Week1/homework/exercise-1/exercise1.js:23]: We deleted the database if it exists, is it still necessary to put a statement for checking? Or is it just for having a good habit for checking?

File: Week1/homework/exercise-2/exercise2.js (hunk 1 above)
Comment (ekmkal) [reply Week1/homework/exercise-2/exercise2.js:15]: Thanks for your time and feedback Cecile🙂

File: Week1/homework/exercise-1/exercise1.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mogaleaf) [reply Week1/homework/exercise-1/exercise1.js:23]: No it was just informative, you don't really need to use it here.

**Pull request closed without merging**